src/_P036_FrameOLED.ino

```diff
@@ -180,9 +189,9 @@ boolean Plugin_036(uint8_t function, struct EventStruct *event, String& string)
 
     case PLUGIN_WEBFORM_LOAD:
     {
-      # ifdef PLUGIN_036_DEBUG
+# ifdef PLUGIN_036_DEBUG
       addLog(LOG_LEVEL_INFO, F("P036_PLUGIN_WEBFORM_LOAD ..."));
-      # endif // PLUGIN_036_DEBUG
+# endif // PLUGIN_036_DEBUG
 
```

Comment:
It is partly a matter of taste, but when enabling the 'Bracket Pairs' feature of VS Code, having intermittent lines starting in column 1 is, IMHO, a bit 'disturbing', but I assume the other formatter changed that, as it once was a convention (or maybe even a requirement?) to have compiler directives starting in column 1.
No need to change 'm back, there are a lot of sources that have that, but Uncrustify doesn't touch them.

![image](https://user-images.githubusercontent.com/1922393/161630472-a6a0360e-79ec-4a52-8917-248f748782b4.png)

So this:
![image](https://user-images.githubusercontent.com/1922393/161630825-c8c6e1c5-a231-486f-a161-02d83c0ef2dd.png)

versus this:
![image](https://user-images.githubusercontent.com/1922393/161630878-835b1e23-01cb-47ce-80ef-d3b969b8d503.png)

Reply:
> No need to change 'm back, there are a lot of sources that have that, but Uncrustify doesn't touch them.

Uncrustify will only apply formatting rules for what is defined.
So you can add it to the uncrustify config if it bothers you :)
Hint: There are online testers to test config in real time.